control/config.txt

```diff
@@ -10,6 +10,10 @@ password
 loginPinCode
 char
 
+# Character Creation Version
+# 1 = Classic Version, 2 = Classic Version without attribute, 3 = Two Sex Account Version , 4 = Doram Version
+charCreateVersion 1
```

Comment:
My idea : add on server.txt

Reply:
@allanon  @lututui  what do you think about?

Reply:
@sctnightcore 
if averyone agree i can move this to servers.txt